<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iconCreationDialog</class>
 <widget class="QDialog" name="iconCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="iconPreviewLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Icon Preview</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="iconPreviewWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <widget class="IconThumbnail" name="iconThumbnail">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="FlatButton" name="browseFolderButton">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../resource/resource.qrc">
                   <normaloff>:/icon/changeDirectory.png</normaloff>:/icon/changeDirectory.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>14</width>
                   <height>16</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>4</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="FlatButton" name="browseFavouriteButton">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../resource/resource.qrc">
                   <normaloff>:/icon/favoriteStar.png</normaloff>:/icon/favoriteStar.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>14</width>
                   <height>16</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget" native="true"/>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="shortcutNameLabel_2">
          <property name="text">
           <string>Command Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QWidget" name="shortcutNameWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="nameEdit"/>
           </item>
           <item>
            <widget class="QCheckBox" name="visibleCheckBox">
             <property name="text">
              <string>visible</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="containingTabLabel_2">
          <property name="text">
           <string>Containing Tab</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QWidget" name="containingTabWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QComboBox" name="containingTabComboBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="FlatButton" name="addTabButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../resource/resource.qrc">
                 <normaloff>:/icon/plus.png</normaloff>:/icon/plus.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Command</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="pythonButton">
          <property name="text">
           <string>Python</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="melButton">
          <property name="text">
           <string>MEL</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CodePlainTextEdit" name="codeTextEdit">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
        <property name="plainText">
         <string/>
        </property>
        <property name="tabStopWidth">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FlatButton</class>
   <extends>QPushButton</extends>
   <header>smartshelf.component.flatbutton.h</header>
  </customwidget>
  <customwidget>
   <class>IconThumbnail</class>
   <extends>QGraphicsView</extends>
   <header>smartshelf.component.iconthumbnail.h</header>
  </customwidget>
  <customwidget>
   <class>CodePlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>smartshelf.component.codeplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>iconCreationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>iconCreationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
